crates/zed/src/languages/ruby/config.toml

```diff
@@ -1,5 +1,5 @@
 name = "Ruby"
-path_suffixes = ["rb", "Gemfile"]
+path_suffixes = ["rb", "Gemfile", "Gemfile.lock", "rake"]
```

Comment:
Is it common to just have `rake` as a filename?

I just spot-checked a few repos (including Rails) and it seems that `Rakefile` is what it's typically called.

Reply:
Yea, see https://guides.rubyonrails.org/command_line.html#custom-rake-tasks for example re `.rake`

Reply:
Got it!

Do you think it's worth adding `Rakefile` as well here, since that seems to be a pretty common name as well?

Reply:
`Gemfile.lock` is associated with Ruby but doesn't use Ruby syntax.